tools/topology/topology1/kernel_dependent/v5.19/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ set(TPLGS_UP
 	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-adl-rt711-l0-rt1316-l13-rt714-l2\;-DHEADSET_DEEP_BUFFER\;-DDYNAMIC=1\;-DPLATFORM=adl\;-DUAJ_LINK=0\;-DAMP_1_LINK=1\;-DAMP_2_LINK=3\;-DEXT_AMP_REF\;-DMIC_LINK=2"
 	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-adl-rt711-l2-rt1316-l01-rt714-l3\;-DHEADSET_DEEP_BUFFER\;-DDYNAMIC=1\;-DPLATFORM=adl\;-DUAJ_LINK=2\;-DAMP_1_LINK=0\;-DAMP_2_LINK=1\;-DEXT_AMP_REF\;-DMIC_LINK=3"
 	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-adl-rt711-l2-rt1316-l01\;-DHEADSET_DEEP_BUFFER\;-DDYNAMIC=1\;-DPLATFORM=adl\;-DUAJ_LINK=2\;-DAMP_1_LINK=0\;-DAMP_2_LINK=1\;-DEXT_AMP_REF\;-DNO_LOCAL_MIC"
+	"sof-icl-rt711-rt1308-rt715-hdmi\;sof-rpl-rt711-l2-rt1316-l01-rt714-l3\;-DHEADSET_DEEP_BUFFER\;-DDYNAMIC=1\;-DPLATFORM=rpl\;-DUAJ_LINK=2\;-DAMP_1_LINK=0\;-DAMP_2_LINK=1\;-DEXT_AMP_REF\;-DMIC_LINK=3"
```

Comment:
this is good but you didn't update the previous patch, it needs 2 lines in two different files as well.

Reply:
@plbossart I'm sorry, I didn't quite understand what you meant.

One of Dell's platforms has no headset. I only need to add the one platform with the headset to the two CMakeLists.txt respectively. Is there something wrong with my understanding?

Reply:
@plbossart 
Do you mean we need to add product configuration in 5.19 CMakeLists.txt even if the product doesn't have headset?
If yes, I think we are missing config for some ADL products in 5.19 CMakeLists.txt.

Reply:
@gongjun-song there are two additions in this 'kernel-dependent' stuff

HEADSET_DEEP_BUFFER is only relevant indeed for platforms with a jack.
DYNAMIC=1 is required for all platforms.

That means that if we are missing ADL products, we need to add them as a correction. We should have exactly the same number of entries for SoundWire devices in tools/topology/topology1/CMakeLists.txt and tools/topology/topology1/kernel_dependent/v5.19/CMakeLists.txt

Can you please double-check if indeed we are missing some devices? Thanks!

Reply:
@plbossart  Thanks for the clarification.
I add a new commit to fix some missing adl topologies in v5.19/CMakeLists.txt. Please help review